### ISSUE TITLE
Can brotab determine which tabs are pinned?

### ISSUE THREAD

**Issue by bjohas** [balta2ar/brotab#116]
Hello,

thanks for building brotab! It's great!

Can brotab work out which tabs are pinned and which ones are not?

Many thanks!

**Comment by doctorcolossus** [comment]
I don't believe so, but I don't think this would be too hard to implement if you want to try and submit a pull request.

In Firefox, [`browser.tabs.query()`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/query) does return a `pinned` boolean which you could use. Surely there is something similar for chromium-based browsers, but you'd have to look into it.

I think the main tricky thing would be exactly how you would envision the exact implementation details and whether others agree, especially the project owner. Like do you imagine returning a boolean value for each tab listed in `bt list`? If so, you'd have to think about aspects like whether that would break backwards compatibility for others, slow anything down, or clutter the list format. Not everyone uses the pinning feature (I don't). Do you also want to support changing tabs' pinned status with `bt update`? Anyway, if you implement something, then others could try it out, give feedback, request changes and suggest fixes, and the owner would probably approve once everything seems good.
